Goal: Do not install the Using Samba book when installing SWAT

Fixes: 

Status wrt upstream: Debian specific

Note: 

Index: samba-3.2.0rc1/source/script/installswat.sh
===================================================================
--- samba-3.2.0rc1.orig/source/script/installswat.sh
+++ samba-3.2.0rc1/source/script/installswat.sh
@@ -198,7 +198,11 @@
 
 # Install/ remove Using Samba book (but only if it is there)
 
-if [ "x$BOOKDIR" != "x" -a -f $SRCDIR../docs/htmldocs/using_samba/toc.html ]; then
+# Under Debian we don't actually install the book. The book is part of
+# the samba-doc package, so we just provide a symlink that points to
+# where the book is actually installed. The symlink is created in
+# debian/rules.
+if /bin/false; then
 
     # Create directories
 
